src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+from typing import Optional
+from redminelib import Redmine
+from enum import Enum
+import errno
+import json
+import requests
+
+from requests.auth import AuthBase
+from mgr_module import CLICommand, CLIWriteCommand, CLIReadCommand, HandleCommandResult
+
+from .. import mgr
+from ..settings import Settings
+from ..exceptions import DashboardException, ScopeNotInRole
+from ..rest_client import RestClient
+from ..model.feedback import FeedbackModel
+
+
+class FeedbackProject(Enum):
```

Comment:
I'd move this enums to the model, as they're intrinsically belong to the data model. In fact, if you do that and `from models import feedback`, then you have the Enums automatically namespaced and don't need to repeat the `Feedback*`, but just `feedback.Severity.minor`.